### PR TITLE
fix: contribute page typos

### PR DESCRIPTION
**Pull request by jdegand**
I removed `or English mistakes` since there are multiple translations now.

### CODE REVIEW

File: docs/src/content/docs/guides/contribute.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ sidebar:
 
 You can contribute to this repository in many ways:
 
-🔥 Create a new challenge by following the intructions [here](/guides/create-challenge).
+🔥 Create a new challenge by following the instructions [here](/guides/create-challenge).
 
 🔥 Answer challenges and submit the results. (guide [here](/guides/resolve-challenge)).
 
-🔥 Comment on other's solutions by providing constructive and caring feedback.
+🔥 Comment on others' solutions by providing constructive and caring feedback.
```
Comment (svenson95) [inline docs/src/content/docs/guides/contribute.md:17]: is that correct? `others'`?

File: docs/src/content/docs/guides/contribute.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ sidebar:
...
 
-🔥 Correct typos or English mistakes within the documentation.
+🔥 Correct typos within the documentation.
```
Comment (svenson95) [inline docs/src/content/docs/guides/contribute.md:19]: > Correct typos within the documentation or add new translations.

Maybe add this "add new translations" part?

File: docs/src/content/docs/guides/contribute.md (hunk 1 above)
Comment (jdegand) [reply docs/src/content/docs/guides/contribute.md]: It is awkward.  `others's` could be right as well.  I changed the line.

File: docs/src/content/docs/guides/contribute.md (hunk 2 above)
Comment (jdegand) [reply docs/src/content/docs/guides/contribute.md:19]: I added a separate line for the translation.  This is a nice addition.

Comment (tomalaforge) [reply docs/src/content/docs/guides/contribute.md:19]: Thanks a lot, that's great 👍

**Review by tomalaforge — APPROVED** [review]

**Pull request merged**